Samples/SqueezeNetObjectDetection/NETCore/cs/Program.cs

```diff
@@ -0,0 +1,172 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+
+using Windows.Storage;
+using Windows.Graphics.Imaging;
+using Windows.Storage.Streams;
+using Windows.AI.MachineLearning;
+using Windows.Foundation;
+using Windows.Media;
+
+using Newtonsoft.Json;
+
+
+namespace SqueezeNetObjectDetectionNC
+{
+    class ImageInference
+    {
+

```

Comment:
remove extra blank lines at the beginning of code block. Also please apply this to rest of the code